upgrade/1.2/scripts/upgrade/ncn-upgrade-ceph-nodes.sh

```diff
@@ -147,7 +147,7 @@ if [[ ${upgrade_ncn} == "ncn-s001" ]]; then
             ssh_keygen_keyscan "${upgrade_ncn}"
             ssh_keys_done=1
         fi
-        scp /usr/share/doc/csm/upgrade/1.0/scripts/upgrade/create_rgw_buckets.sh $upgrade_ncn:/tmp
+        scp /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/create_rgw_buckets.sh $upgrade_ncn:/tmp
```

Comment:
unless we are adding new buckets, do we need this?  it adds time to the upgrade for an idempotent job.  ansible is slow...

Reply:
no new buckets between 1.0 and 1.2 to my knowledge.  good catch.  will remove..